.github/workflows/scripts/lnproxy-sync.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+
+let rawRelays = JSON.parse(fs.readFileSync('./lnproxy_relays.json'));
+let formattedRelays = [];
```

Comment:
There is a chance the `/lnproxy/...relays.json` does not see much action (or stops being maintained altogether). In such case, we might start adding ourselves new relays into `./frontend/static/lnproxies.json` . This script will remove the new proxies we add. Maybe it's best to join existing and incoming proxies instead of starting blank `[]`.

Reply:
OK. I updated the script so that only new proxies that appear in `/lnproxy/...relays.json` will be added to the RoboSats list. We can easily make a more sophisticated merge in the future if needed.